library/test_parse_helm_repositories.py

```diff
@@ -9,18 +9,13 @@ class parse(unittest.TestCase):
     def test_parse_helm_repositories_output(self):
         helm_repositories_output = """
 NAME               	URL
-stable             	https://kubernetes-charts.storage.googleapis.com
```

Comment:
Why are You removing this from the test?

Reply:
Reverted